### PR TITLE
chore: update README

### PR DESCRIPTION
**Pull request by lemeurherve**

### CODE REVIEW

**Review by dduportal — APPROVED** [review]
Great work

File: README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # Help desk
 
-Support for Infrastructure related issues for the Jenkins project
+Support for Infrastructure related issues for the Jenkins project.
 
-As of the 11th of January 2022, all existing Jira INFRA project have been imported in this repository as Github issues.
-See https://github.com/jenkins-infra/helpdesk/issues/9 for more details on the migration. 
+If you want to report an issue or a bug with the Jenkins website, wiki, or any other [services run by the Jenkins project](https://www.jenkins.io/projects/infrastructure/), [open an issue here](https://github.com/jenkins-infra/helpdesk/issues/new/choose)!
```
Comment (timja) [inline README.md:5]: this looks like it was copied from an old version of the website PR?

We removed wiki and website from the text there as neither of them made sense to have an issue opened here?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+
+As of the 11th of January 2022, all existing INFRA project issues have been migrated from [Jira](https://issues.jenkins.io/projects/INFRA/issues) to this repository as [Github issues](https://github.com/jenkins-infra/helpdesk/issues?q=label%3Aimported-jira-issue).
```
Comment (timja) [inline README.md:7]: is this relevant to people? it's very point in time information.

Better to send over in mailing lists than record it I would expect, how many people will want / need this information a week from now.

Comment (lemeurherve) [reply README.md:7]: Agreed on the "point in time", I've put this here for the moment, making it more visible for people clicking on links without reading them or their context, it could be removed next week or even now for what it worth.

File: README.md (hunk 1 above)
Comment (lemeurherve) [reply README.md:5]: Yeah, there is old comment here, probably my mistake, taking back the version suggested by Mark in the previous PR.

Comment (lemeurherve) [reply README.md:5]: It was on another PR: https://github.com/jenkins-infra/jenkins.io/pull/4816#discussion_r783214973
